### ISSUE TITLE
Environment variable setting with type "Set" are interpreted as string

### ISSUE THREAD

**Issue by alim4r** [krateng/maloja#136]
I'm trying to set `MALOJA_REMOVE_FROM_TITLE` in docker to filter `(Explicit)`.

```sh
MALOJA_REMOVE_FROM_TITLE=["(Original Mix)","(Radio Edit)","(Album Version)","(Explicit Version)","(Bonus Track)","(Explicit)"]
```

I think the environment variable is interpreted as `String` and not parsed to an `Array`/`Set`.

https://github.com/krateng/maloja/blob/6171d1d2e1f0c77900c10a290483c08fb73b5ccf/maloja/cleanup.py#L150-L152

In this code snippet the for-loop iterates over every character in the string and removes it from the title if present. For example the characters `O`, `r`, `i`, `g`, `i`, `n`, `a`, `l`.

I used the same syntax in #120 which does work. I assume every other character in the string is also a delimiter now (like `[`, `"`, `]`).
```sh
MALOJA_DELIMITERS_FORMAL=[";","/","|","␝","␞","␟",","]
```

Is my syntax of setting the variable wrong?

**Comment by krateng** [comment]
Environment variables are really only meant just for the base configuration like setting the initial directories - everything more complex than a string should be done with the configuration file. Your second example does indeed make all the characters in that string delimiters.

I see that this is a bit of a problem and should probably indicate failure to the user instead of just doing the wrong thing silently. I can either add a proper data type check or simply disallow setting these things via variable.

**Comment by alim4r** [comment]
Ok thanks! Is the configuration file the `settings.ini` or the `/maloja/pkg_global/conf.py`.

**Comment by krateng** [comment]
`settings.ini` is your configuration file, usually in `/etc/maloja` or in your custom `MALOJA_DATA_DIRECTORY`.

**Comment by krateng** [comment]
Should be fixed in 3.1.1! So the proper syntax should now be:

In the .ini file:
```ini
remove_from_title = ["(Original Mix)","(Radio Edit)","(Album Version)","(Explicit Version)","(Bonus Track)","(Explicit)"]
```

As environment variable (not recommended):
```bash
MALOJA_REMOVE_FROM_TITLE="(Original Mix):(Radio Edit):(Album Version):(Explicit Version):(Bonus Track):(Explicit)"
```
